lib/travis/build/addons/rethinkdb.rb

```diff
@@ -0,0 +1,39 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Rethinkdb < Base
+        SUPER_USER_SAFE = true
+
+        RETHINKDB_GPG_KEY = '0x3A8F2399'
+
+        def after_prepare
+          sh.fold 'rethinkdb' do
+            sh.echo "Installing RethinkDB version #{rethinkdb_version}", ansi: :yellow
+            sh.cmd "service rethinkdb stop", sudo: true
+            sh.cmd "apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 #{RETHINKDB_GPG_KEY}", sudo: true
+            sh.cmd 'add-apt-repository "deb http://download.rethinkdb.com/apt $(lsb_release -cs) main"', sudo: true
+            sh.cmd "apt-get update -qq", assert: false, sudo: true
+            package_version = rethinkdb_version + `lsb_release -cs`
```

Comment:
This is incorrect. <tt>`lsb_releases -cs`</tt> will have to be evaluated at the script's runtime, not the compile time on `travis-build`.

Do something like:

```ruby
sh.cmd "package_version=#{rethinkdb_version}$(lsb_release -cs)"
sh.cmd "apt-get install -y -o Dpkg::Options::='--force-confnew' rethinkdb=$package_version", sudo: true, echo: true, timing: true
```